core/index/field_data.cpp

```diff
@@ -1099,9 +1113,9 @@ bool field_data::invert(token_stream& stream, doc_id_t id) {
     const auto res = terms_.emplace(term->value);
 
     if (nullptr == res.first) {
-      IR_FRMT_ERROR("field '%s' has invalid term of size '" IR_SIZE_T_SPECIFIER "'",
+      IR_FRMT_ERROR("field '%s' contains too long term of size '" IR_SIZE_T_SPECIFIER "'",
```

Comment:
Maybe lower this to "WARNING" ? As user will think something is broken if ERROR appears in log. And add "consequences" description like: "Term will be excluded from the index"